idea/src/META-INF/plugin.xml

```diff
@@ -1592,6 +1592,11 @@
       <category>Kotlin</category>
     </intentionAction>
 
+    <intentionAction>
+      <className>org.jetbrains.kotlin.idea.intentions.IndentRawStringIntention</className>
+      <category>Kotlin</category>
+    </intentionAction>
+
```

Comment:
For future PRs: we have now "branch" files `plugin.xml.172` etc. that also should be modified if you are modifying `plugin.xml`.